runtime/service.go

```diff
@@ -625,7 +632,11 @@ func (s *service) buildVMConfiguration(req *proto.CreateVMRequest) (*firecracker
 	}
 
 	// Create stub drives first and let stub driver handler manage the drives
-	handler, err := newStubDriveHandler(s.shimDir.RootPath(), s.logger, containerCount)
+	handler, err := newStubDriveHandler(
+		string(s.jailer.JailPath()),
```

Comment:
nit: JailPath is returning a vm.Dir which is a string iiuc. Is the string() part needed?

Reply:
Yea, compiler complains otherwise.

Reply:
Is it worth using vm.Dir's RootPath() method that returns a string?

Reply:
I am inclined to say casting seems a little more clear. Casting suggest we are casting the jail path to a string, rather than return the jail path's root path which seems a little confusing.

Reply:
That's fair.